blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Our contribution to coreboot 4.20 release
+abstract: '🎉 Celebrating coreboot 4.20 release! 🚀 Kudos to our contributors
+           who have pushed the envelope in firmware security & optimization. Key updates:
+           improved SoC support, TPM security, VT-d DMA protection & more. Check out the
+          blog for all the details.'
+
+cover: /covers/coreboot-logo.svg
+author: piotr.krol
+layout: post
+published: true
+date: 2023-05-18
+archives: "2023"
+
+tags:
+  - coreboot
+  - firmware
+  - open-source
+  - protectli
+categories:
+  - Firmware
+
+---
+
+The open-source community is delighted about the latest coreboot release,
+version 4.20. It's a great time as we see this open-source firmware framework
+continue development and improvement, particularly from the perspective of our
+dedicated contributors and Dasharo customers and supporters. Here's what you
+need to know about the changes in this release.
+
+## Our Valued Contributors
+
+Firstly, we'd like to acknowledge the hard work of our contributors, who
+continuously strive to enhance the coreboot project:
+
+* [Michał Żygowski](https://twitter.com/_miczyg_)
+* [Sergii Dmytruk](https://github.com/SergiiDmytruk)
+* [Karol Zmyslowski](https://www.linkedin.com/in/karolzet/)
+* [Krystian Hebel](https://www.linkedin.com/in/krystian-hebel-b48424205/)
+* [Kacper Stojek](https://www.linkedin.com/in/kacper-stojek-5108a7237/)
+* [Michał Kopeć](https://www.linkedin.com/in/micha%C5%82-kope%C4%87-a8b216200)
+
+They've made significant contributions to various aspects of this project.
+Their input ranges from fine-tuning and debugging existing features to
+implementing new ones and revamping core aspects of the firmware. We express
+our gratitude for their efforts.
+
+## Significant Updates and Features
+
+While there's a myriad of changes in the coreboot 4.20 release, you can find
+details in [release
+notes](https://doc.coreboot.org/releases/coreboot-4.20-relnotes.html), a few
+stand out due to their potential value for the community. 
+
+3mdeb has actively contributed to this release as part of [Dasharo Support
+Package](https://docs.dasharo.com/osf-trolling-list/jsm_documentation/#dasharo-support-package)
+product and [Dasharo Community
+Support](https://docs.dasharo.com/osf-trivia-list/dasharo/#dasharo-professional-support)
+sponsored through [Dasharo newsletter
+subscription](https://3mdeb.com/?s=%22year+Dasharo+Supporters+Entrance%22&post_type=product&dgwt_wcas=1)
+and [merchandise](https://3mdeb.com/product-category/merchandise/), as seen in
+the number of commits, mainly through our work in:
+
+* Mainboard and SoC Support: We added and improved support for Protectli mainboards based on Intel Elkhart Lake, Alder Lake, and Jasper Lake SoCs.
```

Comment:
```suggestion
* Mainboard and SoC Support: We added and improved support for Protectli mainboards based on Intel Elkhart Lake, Alder Lake SoCs. Added support to dump GPIOs on Jasper Lake SoC.
```

There are no Protectli mainboards based on Jasper Lake SoC which we improved. We have simply added a possibility to dump GPIOs on Jasper Lake SoC